index.html

```diff
@@ -1,51 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link rel="stylesheet" href="./styles.css">
-<script type="text/javascript" src="./js.js"></script>
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <script type="text/javascript" src="./js.js"></script>
+  </head>
+  <body>
     <h1 class="title">Calculator</h1>
-<div class="calc-background">
-    <section class="calc-row">
+    <div class="calc-background">
+      <section class="calc-row">
         <p class="answer-number">0</p>
-    </section>  
+      </section>
 
-   <section class="calc-row">
-    <p class="half-button grey-button">C</p>
-    <p class="quarter-button grey-button">&larr;</p>
-    <p class="quarter-button orange-button">%</p>
-   </section>
+      <section class="calc-row">
+        <p class="half-button grey-button">C</p>
+        <p class="quarter-button grey-button">&larr;</p>
+        <p class="quarter-button orange-button">%</p>
+      </section>
 
-   <section class="calc-row">
-    <p class="quarter-button grey-button">7</p>
-    <p class="quarter-button grey-button">8</p>
-    <p class="quarter-button grey-button">9</p>
-    <p class="quarter-button orange-button">x</p>
-   </section>
+      <section class="calc-row">
```

Comment:
HTML section is not used semantically correct here. I suggest using an unordered list.